File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.js
class Solution {
    longestConsecutive(nums) {
        let seqSet = new Set()
        let maxSeq=0
        nums.forEach((item)=>{
            seqSet.add(item)
        })
        nums.forEach((item)=>{
            if(!seqSet.has(item-1)){
               let currentSeq=1;
               let num=item
               while(seqSet.has(num+1)){
                currentSeq++
                num++
               }
               if(currentSeq>maxSeq){
                maxSeq=currentSeq
               }
            }
        })
        return maxSeq
    }
}

File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        i,n,=0,len(nums)
        while i<n:
            num=nums[i]
            if num==val:
                n -= 1
                nums[i] = nums[n]
            else:
                i+=1
        return n